src/js/injection_script.js

```diff
@@ -2619,6 +2622,17 @@ domainBypass('apkadmin.com', () => {
   })
 })
 
+domainBypass("bowfile.com", () => {
+	const regex=/.*let next = "(http[^"]+)";.*/
+	document.querySelectorAll("script").forEach(script=>{
+		let matches=regex.exec(script.textContent)
+		if(matches&&matches[1])
+		{
+			unsafelyNavigate(matches[1])
```

Comment:
Did you specifically choose for unsafelyNavigate?

Reply:
Cause it can add a wait before sending u further which was required for this case

Reply:
Use safelyNavigate, it does the same thing after checking the url scheme

Reply:
> Use safelyNavigate, it does the same thing after checking the url scheme

Alright, didn't know this as I'm new to contributing to the extension, I've changed all my usages to safelyNavigate now.

Reply:
I am actually working on rebuilding this so in the future you could just do: FastForward.redirect({URL, wait_time}) etc

Reply:
> I am actually working on rebuilding this so in the future you could just do: FastForward.redirect({URL, wait_time}) etc

That would be pretty nice to be honest as currently it's pretty annoying how you got to add a case to some switch case if you require a wait time for your situation